getting_started/installation.adoc

```diff
@@ -10,67 +10,135 @@
 
-*Installing and Starting the Nodes*
+*Installing and Running an All-in-One Server*
 
-Configure each node host with the appropriate yum repository for OpenShift. Then, install the [package]#openshift-node# package:
+. Download and untar the binary from the https://github.com/openshift/origin/releases[Releases] page on your local system.
 
+. Launch the server:
++
 ----
-# yum install openshift-node
+$ sudo ./openshift start
 ----
++
+This command:
++
+* starts OpenShift listening on all interfaces (https://0.0.0.0:8443),
+* starts the Management Console listening on all interfaces (https://0.0.0.0:8444),
+* launches an [sysitem]#etcd# server to store persistent data, and
+* launches the Kubernetes system components.
++
+The server runs in the foreground until you terminate the process.
++
+NOTE: This command requires `root` access to create services due to the need to modify `iptables`. See https://github.com/GoogleCloudPlatform/kubernetes/issues/1859[this Issue] for more information.
 
-NOTE: Currently, nodes must be started as the root user so the Kubernetes proxy can manipulate the iptables rules to expose service ports. However, the link:../using_openshift/cli.html[OpenShift CLI] does not need to be run as root.
-
-To start the nodes, run the `openshift start node` command on each node host. Set the `--master` option to the host name of the master.
+. You can see more about the commands available in the binary with:
++
+----
+$ ./openshift help
+----
 
+. Because OpenShift services are secured by TLS, clients must accept the server certificates and present their own client certificate. These certificates are generated when the master server is started. You must point `osc` and `curl` at the appropriate CA bundle and client key and certificate to connect to OpenShift. Set the following environment variables:
++
 ----
-# openshift start node --master=<master_IP>
+$ export KUBECONFIG=`pwd`/openshift.local.certificates/admin/.kubeconfig
+$ export CURL_CA_BUNDLE=`pwd`/openshift.local.certificates/admin/root.crt
+$ sudo chmod +r `pwd`/openshift.local.certificates/admin/key.key
 ----
++
+NOTE: This is just for example purposes; in a production environment, developers would generate their own keys and not have access to the system keys.
 
-The server runs in the foreground until you terminate the process.
+. You can see more about the commands available in the CLI with:
++
+----
+$ ./osc help
+----
++
+Or connect from another system with:
++
+----
+$ ./osc -h <server_hostname_or_IP> [...]
+----
 
 *What's Next?*
 
-Now that you have a master and nodes running in your OpenShift environment, link:try_it_out.html[try it out] by walking through a sample application lifecycle.
+Now that you have OpenShift successfully running in your environment, link:try_it_out.html[try it out] by walking through a sample application lifecycle.
+endif::[]
+
+ifdef::openshift-origin[]
```

Comment:
Thanks, I wasn't sure if it was better to combine blocks in these situations versus keeping them separated, in case the sections moved around or were conditionalized differently in the future. I can combine them.